src/nupic/encoders/PassThroughEncoder.hpp

```diff
@@ -0,0 +1,208 @@
+/* ---------------------------------------------------------------------
+* Numenta Platform for Intelligent Computing (NuPIC)
+* Copyright (C) 2014-2016, Numenta, In  Unless you have an agreement
+* with Numenta, In, for a separate license for this software code, the
+* following terms and conditions apply:
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero Public License version 3 as
+* published by the Free Software Foundation.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+* See the GNU Affero Public License for more details.
+*
+* You should have received a copy of the GNU Affero Public License
+* along with this program.  If not, see http://www.gnu.org/licenses.
+*
+* http://numenta.org/licenses/
+* ---------------------------------------------------------------------
+*/
+
+#ifndef NTA_passthroughencoder_HPP
+#define NTA_passthroughencoder_HPP
+
+#include <algorithm>
+#include <iostream>
+#include <string> 
+#include <vector>
+#include <tuple>
+#include <map> 
+
+#include <nupic/encoders/Encoder.hpp>  
+#include <nupic/encoders/Utils.hpp>
+#include <nupic/encoders/RangeList.hpp>
+#include <nupic/encoders/DecodeResult.hpp>  
+
+using namespace std;
+
+namespace encoders {
+
+	/**
+	* Pass an encoded SDR straight to the model.
```

Comment:
I think we should drop this encoder since it does nothing.  The things which this does do are already part of the SDR class.

Reply:
I was hesitating to do that, as an identity encoder might be useful for bootstraping new encoders, or for its "decode" feature. 
But you are probably right, I'll remove it and open an issue on Decoding.